day-range-picker/styles/day-range-picker.css

```diff
@@ -0,0 +1,282 @@
+@charset "UTF-8";
+
+.daypicker-range {
+  --daypicker-primary-color: #ffffff;
```

Comment:
I would expose these CSS variables in the documentation. Like, "these are the CSS variables you can set namespaced to `.daypicker-range` to control the style and look and feel of the calendar"